djangocms_blog/models.py

```diff
@@ -232,7 +232,7 @@ def get_title(self):
         return title.strip()
 
     def get_keywords(self):
-        return self.safe_translation_getter('meta_keywords').strip().split(',')
+        return self.safe_translation_getter('meta_keywords', default='').strip().split(',')
```

Comment:
@fmarco Now that is interesting because I have all migrations applied and yet meta_keywords is not null with no default in database. Actually all field in ```djangocms_blog_post_translation``` are not null with no default... So either there were some migration changes after they were first created or it has to do with parler. I have to investigate this a bit.